src/components/ProjectList/ProjectListContainer/index.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 
-import { rtd } from './projectListContainer.module.scss';
+import { rtc } from './projectListContainer.module.scss';
 
-const ProjectListContainer = ({ children }) => (
-  <table>
-    <thead>
-      <tr>
-        <th>Project Health</th>
-        <th>Project Name</th>
-        <th className={rtd}>Last Updated</th>
-      </tr>
-    </thead>
-    <tbody>{children}</tbody>
-  </table>
-);
+const ProjectListContainer = ({ children, status }) => {
+  const statusTitleArr = [];
+  var i;
+  if (status.length > 0) {
+    for (i = 0; i < 4 && !(i >= status.length); i++) {
+      statusTitleArr.push(status[i]);
+    }
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Project Name</th>
+            {/* <th className={rtd}>Last Updated</th> */}
```

Comment:
If commented code is no longer relevant, it can be removed from code base to tidy up

Reply:
I left this in because I wasn't sure if external integrations was still planning on using it. If not, we planned on using it to display how recently labels/columns were updated. I know this goes against conventions on commented code, but I thought I'd leave it in until we spoke with the other teams about it. I can rip it out and store it in another branch if it's too much of a problem in master, though.

Reply:
It may be relevant to our team and the others. We're just not using it at the moment. Jess wanted us to take it out until then. I'll be sure to drop a comment in the code explaining it until that issue is resolved. Thanks for the feedback!